mergify_engine/worker.py

```diff
@@ -884,6 +809,23 @@ async def _consume_pull(
             logger.error("failed to process pull request", exc_info=True)
             raise UnexpectedPullRetry()
 
+    async def should_handle_owner(
+        self,
+        owner_id: github_types.GitHubAccountIdType,
+        dedicated_worker_owner_ids: typing.Set[github_types.GitHubAccountIdType],
```

Comment:
I find it odd that the StreamProcessor does not catch this itself since it's the main user of it.

Reply:
We have 30 StreamProcessor per process and don't want to duplicate the set or sync 30 times the set.